pkg/controller/certificates/certificate_controller_test.go

```diff
@@ -64,7 +64,8 @@ func TestCertificateController(t *testing.T) {
 
 	stopCh := make(chan struct{})
 	defer close(stopCh)
-	go informerFactory.Start(stopCh)
+	informerFactory.Start(stopCh)
+	informerFactory.WaitForCacheSync(stopCh)
```

Comment:
Just because the cache has synced, does that necessarily mean all the notifications have made it to the handler function that adds an item to the work queue? I thought those were called in go routines as well